providers/apache/pinot/src/airflow/providers/apache/pinot/hooks/pinot.py

```diff
@@ -295,7 +295,7 @@ class PinotDbApiHook(DbApiHook):
 
     def get_conn(self) -> Any:
         """Establish a connection to pinot broker through pinot dbapi."""
-        conn = self.get_connection(self.pinot_broker_conn_id)  # type: ignore
+        conn = self.get_connection(self.pinot_broker_conn_id)
```

Comment:
Had the same problem in `vertica` --> you need to switch to
```suggestion
        conn = self.get_connection(self.get_conn_id())
```